packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js

```diff
@@ -132,9 +138,9 @@ export function getDisplaySrcFromFontFace( input, urlPrefix ) {
 	} else {
 		src = input;
 	}
-	// If it is a theme font, we need to make the url absolute
-	if ( src.startsWith( 'file:.' ) && urlPrefix ) {
-		src = src.replace( 'file:.', urlPrefix );
+	// If it is a theme we expect the font face be already loaded in the browser.
```

Comment:
WordPress tries to avoid self-referential "we" comments.

```suggestion
	// It's expected theme fonts will already be loaded in the browser.
```